src/components/image/index.tsx

```diff
@@ -52,6 +53,20 @@ function getContentApiDimensions(
 	return null
 }
 
+function getTheme(src: string): GlobalThemeOption | undefined {
+	let theme
+	const url = new URL(src, 'https://developer.hashicorp.com')
+	const themedImageSuffix = new RegExp(/#hide-on-(dark|light)/)
```

Comment:
(Opinion): I think reversing the logic here will be more user-friendly

```diff
// instead of saying "I want to hide this lightmode image on dark mode"
- #hide-on-dark
// say "I want to only show this darkmode image on dark mode"
+ #dark-only
```

what do you think?

Reply:
💡 yeah now that you mention it, I see how that could be confusing. I think my brain just defaulted to the way we use the `data-hide-on-{theme}` property. What do you think of 
```
#{dark|light}-theme-only
```
To be explicit that this is related to theming?

Reply:
> ```
> #{dark|light}-theme-only
> ```
> To be explicit that this is related to theming?

💯 I like this a lot!

Reply:
Updated!